src/Persistent/Api.hs

```diff
@@ -308,8 +313,7 @@ instance FromProto IdeaLike where
     fromProto () = IdeaLike
 
 -- FIXME: Same user can like the same idea more than once.
--- Assumption (check when enforceDebuggingAssertions):
--- the given @AUID Idea@ MUST be in the DB.
+-- FIXME: Assumption: the given @AUID Idea@ MUST be in the DB.
```

Comment:
because they should be assertions?  perhaps that was a misunderstanding, and we don't want to / can't assert this?